File: CMOWebApi.Models/AdminModel/MasterModel/HelpDocumentTypeMasterModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CMOWebApi.Models.AdminModel.MasterModel
{
	public class HelpDocumentTypeMasterModel
	{
		public long Id { get; set; }
		public Nullable<long> Code { get; set; }
		public string Name { get; set; }
		public string NameHindi { get; set; }
		public Nullable<bool> IsActive { get; set; } = true;
		public Nullable<bool> IsDelete { get; set; } = false;
		public Nullable<System.DateTime> CreatedDate { get; set; }
		public Nullable<int> CreatedBy { get; set; }
		public Nullable<System.DateTime> ModifiedDate { get; set; }
		public Nullable<int> ModifiedBy { get; set; }
		public string SSOID { get; set; }
		public Nullable<bool> IsShowInWebServiceMaster { get; set; }

	}
}

File: ANGULAR/ANGULAR/src/app/Shared/Model/help-document-type-master-model.ts
export class HelpDocumentTypeMasterModel {
  Id: number;
  Code: number | null;
  Name: string;
  NameHindi: string;
  IsActive: boolean | null;
  IsDelete: boolean | null;
  CreatedDate: Date | string | null;
  CreatedBy: number | null;
  ModifiedDate: Date | string | null;
  ModifiedBy: number | null;
  SSOID: string;
	IsShowInWebServiceMaster: boolean | null;
}
